bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Decryption/DecryptWithSecrets.cs

```diff
@@ -7,6 +7,14 @@ namespace ElectionGuard.Decryption.Decryption;
 /// </summary>
 public static class DecryptWithSecretsExtensions
 {
+    // when decrypting with secrets, we do not currently have a proof
+    [Obsolete("This method is obsolete and kept for testing purposes only when decrypting with secret values during unit testing. Use the overload with a proof.")]
+    public static void Update(this PlaintextTallySelection self, ulong tally)
+    {
+        using var value = BigMath.PowModP(Constants.G, tally);
+        self.Update(tally, value, self.Proof);
+    }
+
```

Comment:
Should the Decrypt methods that use this Obsolete method also be marked as Obsolete?